.github/workflows/ci-cygwin-standard.yml

```diff
@@ -59,6 +59,7 @@ jobs:
     needs: [cygwin-stage-i-a, cygwin-stage-i-b]
 
   cygwin-stage-ii-e:
+    uses: ./.github/workflows/cygwin.yml
     env:
```

Comment:
I think the `env` block here also need to be converted into `with`

Reply:
Thanks, I've updated that.